packages/frontend/app/logging.ts

```diff
@@ -36,9 +37,15 @@ configure({
             layout: { type: 'json', separator: ',' },
         },
     },
-    categories: { default: { appenders: ['fileAppender'], level: 'info' } },
+    categories: {
+        default: { appenders: ['fileAppender'], level: 'info' },
+        development: { appenders: ['console'], level: 'warn' },
+    },
     pm2: true,
 });
 
-export const logger = getLogger();
+export const logger =
+    process.env.NODE_ENV === 'development'
+        ? getLogger('development')
+        : getLogger();
 logger.level = 'info';
```

Comment:
Updated the logger to only print to console in the development environment so that any validation errors are clearly logged.  This is behaviour we can remove or change later  as preferred.